CONTRIBUTING.md

```diff
@@ -0,0 +1,119 @@
+# How to contribute to Eclipse Hono
```

Comment:
I guess this should be SDV Blueprints or Fleet Management in SDV Blueprints

Reply:
right, thanks for uncovering my shameless copy/paste approach ... ;-)